### PR TITLE
Update/vep102

### PR DESCRIPTION
**Pull request by matthdsm**

### CODE REVIEW

File: config/genomes/GRCh37-resources.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-version: 54
+version: 55
 
 aliases:
   human: true
   snpeff: GRCh37.75
-  ensembl: homo_sapiens_merged_vep_100_GRCh37
+  ensembl: homo_sapiens_merged_vep_103_GRCh37
```
Comment (roryk) [inline config/genomes/GRCh37-resources.yaml:6]: Is 103 here a typo?

File: config/genomes/GRCz11-resources.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-version: 7
+version: 8
 
 aliases:
-  ensembl: danio_rerio_vep_100_GRCz11
+  ensembl: danio_rerio_vep_103_GRCz11
```
Comment (roryk) [inline config/genomes/GRCz11-resources.yaml:4]: 103

**Review by roryk — COMMENTED** [review]
Thanks Matthias for keeping these up to date and adding the `--mane` option, I think there are a couple of typos 103 should be 102.

**Comment by roryk** [comment]
Don't worry about the failed checks, our docker image is borked right now.

File: config/genomes/GRCz11-resources.yaml (hunk 2 above)
Comment (matthdsm) [reply config/genomes/GRCz11-resources.yaml]: indeed, that's a typo!

**Comment by matthdsm** [comment]
typo's fixed!

M

**Comment by naumenko-sa** [comment]
Thanks, @matthdsm !

**Pull request merged**